swan-lake/learn-the-platform/ballerina-tooling/ballerina-shell.md

```diff
@@ -288,13 +289,11 @@ For example, see below.
 ```
 =$ /help strings
 | 
-| Topic description :
+| # Strings
 | 
-|  The `string` type represents immutable sequence of zero or more Unicode characters. 
-|  There is no separate character type: a character is represented by a `string` of length 1.
-|  Two `string` values are `==` if both sequences have the same characters.
-|  You can use `<`, `<=`, `>`, and `>=` operators on `string` values and they work by comparing code points.
-|  Unpaired surrogates are not allowed.
+| The `string` type represents immutable sequence of zero or more Unicode characters.  There is no separate character type: a character is represented by a `string` of length 1.
+| 
+| Two `string` values are `==` if both sequences have the same characters. You can use `<`, `<=`, `>`, and `>=` operators on `string` values and they work by comparing code points. Unpaired surrogates are not allowed.
```

Comment:
```suggestion
| Two `string` values are `==` if both sequences have the same characters. You can use `<`, `<=`, `>`, and `>=` operators on `string` values and they work by comparing the code points. Unpaired surrogates are not allowed.
```

Reply:
same as above comment

Reply:
The BBE source is fixed via https://github.com/ballerina-platform/ballerina-distribution/pull/3340.